### ISSUE TITLE
Define GTM tracker

### ISSUE THREAD

**Issue by mazipan** [kawalcovid19/wargabantuwarga.com#507]
## Description

GTM have ability to handle the event tracker and sending it to other analytic tools like GA.

We need to start define which part that we want to track.

## Implementation Model

Create a centralized doc first to collect all the event to be tracked (maybe use Google Sheet), list down all the events, then we can distribute to the developer.

**Comment by zainfathoni** [comment]
Mas @baraeb92, can we rely on you for preparing the document? 😁 
Thanks! 🙏

**Comment by baraeb92** [comment]
Yes, I'll try to identify which flow in our web that might need additional work or datalayer to be track properly. 

Most click has been recorded at the moment, so there might not be a lot of custom datalayer needed, the only one that I can think of is search, where we might have to record user keyword input. 

But let me double check on that. @zainfathoni
